### DISCUSSION TITLE
如何在本项目实时预览 lobe-ui 修改

### DISCUSSION THREAD

**Discussion by sxjeru** [lobehub/lobehub#2398] [Development Question]
这边可能需要向 lobe-ui 提交更改，但不知如何将修改应用到本项目上，以便预览效果。

或许实时的可能性不大，只需能在此项目应用 ui fork 库的更改即可。

尝试过 package.json git+xxx，不成。

**Comment by arvinxx** [comment]
我们有一个私有的基于 submodule 的 monorepo ，可以实现这个效果，近期有空我们把它开源出来。
